go/stellar/stellarsvc/service.go

```diff
@@ -206,3 +206,21 @@ func (s *Server) WalletInitLocal(ctx context.Context) (err error) {
 	_, err = stellar.CreateWallet(ctx, s.G())
 	return err
 }
+
+func (s *Server) WalletGetPublicKeys(ctx context.Context) (ret []stellar1.BundleEntry, err error) {
```

Comment:
This use of `BundleEntry` I don't like. How about `GetOwnAccountIDs(ctx,g) -> ([]stellar1.AccountID, error)`?